### PR TITLE
Add lidar sensor delay

### PR DESCRIPTION
**Pull request by kosuke55**
## Types of PR

- [x] New Features
- [ ] Upgrade of existing features
- [ ] Bugfix

## Description
similar to https://github.com/tier4/scenario_simulator_v2/pull/964, add lidar sensor pointcloud delay.

https://github.com/tier4/scenario_simulator_v2/assets/39142679/2b8dad25-bb41-4add-b81f-3a9eb5f0fc40

[tier4 internal ticket](https://tier4.atlassian.net/browse/RT1-2876)

### CODE REVIEW

**Comment by HansRobo** [comment]
This pull-request is waiting for #998.
@kosuke55 After closing #998, could you resolve conflicts and do the regression test?

**Comment by kosuke55** [comment]
Add sample scenario in https://github.com/tier4/scenario_simulator_v2/pull/1023/commits/76d3ed24d12effe0f5f295ad08a2e5d768f5d7dc

Screenshot from 2023-06-29 12-53-48

https://github.com/tier4/scenario_simulator_v2/assets/39142679/333e4b8c-1f28-4326-976a-c76ebf6ad3cc

File: docs/developer_guide/ConfiguringPerceptionTopics.md [hunk 1]
```diff
@@ -190,6 +191,51 @@ it reaches the planning module.
...
+
+**Note** - This feature only adjusts the interval between ssv2 generating a
+object pointcloud and publishing it. Note that there is another kind of delay
+between when `scenario_simulator_v2` publishes the perception result and when
+it reaches the planning module.
+
+**Default behavior** - If the property is not specified, the default value is
+`"0.0"`, meaning no delay.
+
+**[Example](https://github.com/tier4/scenario_simulator_v2/blob/master/test_runner/scenario_test_runner/scenario/Property.pointcloudObjectPublishingDelay.yaml)** -
```
Comment (HansRobo) [inline docs/developer_guide/ConfiguringPerceptionTopics.md:226]:
```suggestion
**[Example](https://github.com/tier4/scenario_simulator_v2/blob/master/test_runner/scenario_test_runner/scenario/Property.pointcloudPublishingDelay.yaml)** -
```

File: docs/developer_guide/ConfiguringPerceptionTopics.md [hunk 2]
```diff
@@ -128,6 +129,39 @@ not supported.
...
+
+**Specification** - For each frame, if the current stamp exceeds the set time
+from the topic's stamp, the topic is published, thereby implementing the delay.
+This could result in a maximum error of 2/FPS. Also, the order of topics to be
+published is guaranteed to be the same as when there is no delay.
+Since the delay is set to the same value for each object, it is not possible to
+delay only a specific object.
+
+**Note** - This feature sets the delay from when the perception results are
+obtained in `scenario_simulator_v2` to when they are published. Not that there
```
Comment (kyoichi-sugahara) [inline docs/developer_guide/ConfiguringPerceptionTopics.md:157]: not but Note?

File: docs/developer_guide/ConfiguringPerceptionTopics.md [hunk 3]
```diff
@@ -128,6 +129,39 @@ not supported.
...
+are other potential sources of delay, such as frame rates becoming unstable,
+and delays in pub/sub.
+
+**Default behavior** - If the property is not specified, the default value is
```
Comment (kyoichi-sugahara) [inline docs/developer_guide/ConfiguringPerceptionTopics.md:161]: how about "is not fefined in the scenario file,"

File: docs/developer_guide/ConfiguringPerceptionTopics.md [hunk 4]
```diff
@@ -128,6 +129,39 @@ not supported.
...
+`"0.0"`, meaning no delay.
```
Comment (kyoichi-sugahara) [inline docs/developer_guide/ConfiguringPerceptionTopics.md:162]: indicating no delay?

File: docs/developer_guide/ConfiguringPerceptionTopics.md (hunk 2 above)
Comment (kosuke55) [inline docs/developer_guide/ConfiguringPerceptionTopics.md:157]:
```suggestion
**Note** - This feature sets the delay from when the perception results are
obtained in `scenario_simulator_v2` to when they are published. Note that there
```

File: docs/developer_guide/ConfiguringPerceptionTopics.md (hunk 4 above)
Comment (kosuke55) [inline docs/developer_guide/ConfiguringPerceptionTopics.md:162]:
```suggestion
`"0.0"`, indicating no delay.
```

**Review by HansRobo — APPROVED** [review]
LGTM, waiting for
- [ ] importing master after closing #998, and resolving conflicts
- [ ] regression test on Autoware Evaluator

**Comment by kosuke55** [comment]
waiting for
- [ ] https://github.com/tier4/scenario_simulator_v2/pull/1028

File: docs/developer_guide/ConfiguringPerceptionTopics.md (hunk 3 above)
Comment (kosuke55) [reply docs/developer_guide/ConfiguringPerceptionTopics.md:161]:
```suggestion
**Default behavior** - If the property is not defined, the default value is
```

Comment (kosuke55) [reply docs/developer_guide/ConfiguringPerceptionTopics.md:161]: thanks

**Comment by kosuke55** [comment]
@HansRobo 
I checked degradation and it was ok!
https://github.com/tier4/sim_evaluation_tools/issues/160

**Pull request merged**